Winium/Winium.StoreApps.InnerServer/Element/WiniumElement.IsUserVisible.cs

```diff
@@ -82,6 +89,42 @@ internal bool IsUserVisible()
             }
         }
 
+        private CommandBar FindCommandBar(AppBarButton element)
+        {
+            DependencyObject parent = element;
```

Comment:
Not sure what you are trying to accomplish here. But I could probably shorten it a bit by using `do while()` and skipping one of the varialbes. ( Just have one `FrameworkElement` instead of one `FrameworkElement` and one `DependencyObject` )

PS: three ``` for multi line code sections : )